### DISCUSSION TITLE
Add more widget into Overview

### DISCUSSION THREAD

**Discussion by huyhoang8398** [allure-framework/allure2#2750] [Questions & Support]
![image](https://github.com/user-attachments/assets/2545bfeb-edf6-4aaa-86d8-f481a38b6c4d)
Is this possible to add more widget to the overview section?

**Comment by baev** [comment]
Could you please elaborate on what data you'd like to display?

**Reply by huyhoang8398** [reply]
I would like to add more widget ( same as environment ) since I have a lots of environment datas to display and it's might be better if i can split it into smaller sub section
